zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -321,34 +322,20 @@ impl ReadDisk for DiskDb {
     fn zs_first_key_value<C, K, V>(&self, cf: &C) -> Option<(K, V)>
     where
         C: rocksdb::AsColumnFamilyRef,
-        K: FromDisk,
+        K: IntoDisk + FromDisk,
```

Comment:
This IntoDisk bound is no longer needed. (Same for all the methods that use seek.)